### ISSUE TITLE
`deno-lint-ignore-file` comment conflicts with shebang

### ISSUE THREAD

**Issue by AnInternetTroll** [denoland/deno_lint#999]
### Lint Name
deno-lint-ignore-file

### Code Snippet

```ts
#!/usr/bin/env -S deno run --no-check
// deno-lint-ignore-file

const variable = "I am useless!";

console.log("Variables? Who needs em");
```

### Expected Result
no error
### Actual Result
```ts
(no-unused-vars) `variable` is never used
const variable = "I am useless!";
      ^^^^^^^^
    at /home/luca/test.ts:4:6

    hint: If this is intentional, prefix it with an underscore like `_variable`
    help: for further information visit https://lint.deno.land/#no-unused-vars

Found 1 problem
Checked 1 file
```

### Version
```console
deno 1.17.3+9def449 (canary, x86_64-unknown-linux-gnu)
v8 9.8.177.2
typescript 4.5.2
```
